### PR TITLE
New changelog entry for v3.6.2

### PR DESCRIPTION
**Pull request by garethx**
- [x] Testing
- [x] Ready to be merged

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3917/commits/dc96bde457fa377028a02854f18d7d6f88891529 is at https://pr3917.build.csb.dev/s/new.

**Comment by CompuIves** [comment]
One more bug fix: "We now support showing type definitions from dependencies that have git dependencies". This fixes types from @codesandbox/components as well.

File: packages/homepage/content/changelog/3-6-2/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+#### Features
+
+1. You can now highlight a selection of code when sharing a sandbox URL
+2. We no longer show ads in the editor to anonymous users
+3. [@jyash97](https://github.com/codesandbox/codesandbox-client/pull/3886) added
+   a copy sandbox link option for sandboxes on the dashboard
+
+#### Fixes
+
+- We introduced a new way of loading dependencies reducing first-time install
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-6-2/index.md:20]: Shouldn't this be a feature? 🤔

File: packages/homepage/content/changelog/3-6-2/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  times by up to 60x and 4x when cached
+- Lots of tweaks to Live making it handle edge-cases more reliably
+- We made some performance improvements that make VSCode load faster
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-6-2/index.md:23]: Same

File: packages/homepage/content/changelog/3-6-2/index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+- VSCode user snippets and extension quick input commands work again
+- We’ve updated the TypeScript version in the editor from 3.6 to 3.8
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-6-2/index.md:25]: Same

File: packages/homepage/content/changelog/3-6-2/index.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+- Upgraded the type fetching service to handle more concurrent requests and type more
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-6-2/index.md:26]: Same

File: packages/homepage/content/changelog/3-6-2/index.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  dependencies
+- We now display the correct loading state for buttons when forking
+- Fixed running Jest tests with JSDOM
+- Setting custom VSCode themes now works again
+- We now show the reason why an import has failed rather than just the error
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-6-2/index.md:31]: Same

File: packages/homepage/content/changelog/3-6-2/index.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+  code
+- Fixed-up an error message when opening markdown files
+- We added a button when viewing previews fullscreen to protect folks from
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-6-2/index.md:34]: Same

File: packages/homepage/content/changelog/3-6-2/index.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+  phishing attempts
+- Fixed bundler issues resolving Sass imports and importing CSS from npm
+  dependencies
+- We now show the anonymous sandbox limit error if it’s reached when forking
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-6-2/index.md:38]: Same

File: packages/homepage/content/changelog/3-6-2/index.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+  from the “Create New Sandbox” screen
+- Fixed an issue which prevented changing modes in a live session
+- [@MichaelDeBoey](https://github.com/codesandbox/codesandbox-client/pull/3888)
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-6-2/index.md:41]: Same

File: packages/homepage/content/changelog/3-6-2/index.md (hunk 1 above)
Comment (garethx) [reply packages/homepage/content/changelog/3-6-2/index.md:20]: No as it's not a user facing feature

File: packages/homepage/content/changelog/3-6-2/index.md (hunk 2 above)
Comment (garethx) [reply packages/homepage/content/changelog/3-6-2/index.md:23]: No, same reason

File: packages/homepage/content/changelog/3-6-2/index.md (hunk 4 above)
Comment (garethx) [reply packages/homepage/content/changelog/3-6-2/index.md:26]: Same

File: packages/homepage/content/changelog/3-6-2/index.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
+---
+banner:
+title: v3.6.2
```
Comment (MichaelDeBoey) [inline packages/homepage/content/changelog/3-6-2/index.md:3]: Shouldn't this be `3.7.0`, since it adds features? 🤔

Comment (SaraVieira) [reply packages/homepage/content/changelog/3-6-2/index.md:3]: As marked these are not considered featured

**Pull request merged**